AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2721,6 +2728,11 @@ - (ASEnvironmentTraitCollection)environmentTraitCollection
   return _environmentState.traitCollection;
 }
 
+- (void)setEnvironmentTraitCollection:(ASEnvironmentTraitCollection)environmentTraitCollection
```

Comment:
The naming seems a bit inconsistent - we have both asyncTraitCollection and environmentTraitCollection.  Each name seems great in the situation it is used, but we should consider if a quick refactoring can make this any clearer.

The line above has a similar issue.  layoutSpec.traitCollection is of a different type than node.traitCollection.